src/docs/product/accounts/membership.mdx

```diff
@@ -9,87 +9,116 @@ description: "Learn about the different types of Sentry membership, roles, and a
+| Can invite new members to join the organization (_Invite will land in pending state for Managers and Owners to approve_) | ✓                    | ✓                                                                    |
+| Can remove an existing team from the organization                                                                        |                      | ✓ (_Only for teams they’re an existing admin for_)                   |
+| Can transer existing projects                                                                                            |                      | ✓ (_Only for projects owned by teams they’re an existing admin for_) |
+| Can remove existing projects from the organization                                                                       |                      | ✓ (_Only for projects owned by teams they’re an existing admin for_) |
+| Can create new teams for the organization                                                                                |                      |                                                                      |
+| Can create new projects for the organization                                                                             |                      |                                                                      |
 
-To restrict Team access, go to Organization Settings and flip the “Open Membership” toggle.
+### Edge Cases
+
+In cases where several roles apply to one resource, a union of all permissions is applied.
+
+Lets say a user is a Team Admin for Team 1 and a Contributor for Team 2, and both teams own Project A. The user can:
+
+- View and act on issues in Project A.
+- Change settings for Project A.
+- Add and remove Contributors for Team 1, but not for Team 2.
+- Remove Project A from Team 1, but not from Team 2.
+
+## Project Access and Team Membership
+
+Project access is limited to that project's team(s). Team Admins, Org Managers, and Org Owners can add or remove project teams in **Project Settings > Project Teams**. Alternatively, these roles can also add projects to a team in **Team Settings > Projects**.
+
+All Org Members can get access to a project by joining a team associated with the project.
+
+### Open Membership
+
+By default, team membership is open. With "Open Membership" enabled, all members can join, leave, and add other members to teams.
+
+To restrict access to teams and projects, turn off "Open Membership" for the organization in **Settings > General Settings**.
 
 ![](membership-toggle.png)
 
-## Transferring a Project
+When "Open Membership" is toggled off, teams are invite-only. Users can only join a team if they're invited by an Org Manager, Org Owner, or a Team Admin for that team. In this case, Org Admins who are not part of the team also need an invite to join. After they join, they automatically become a Team Admin.
+
+## Transfer a Project
```

Comment:
I tried to do this in my personal org, and it seems like you can only transfer projects to another Org Owner. Can we get additional details on this?

Reply:
Fixed here: https://github.com/getsentry/sentry/pull/51557